openapi-issuer.yaml

```diff
@@ -108,6 +108,31 @@ paths:
           $ref: "#/components/responses/NotFound"
         default:
           $ref: "#/components/responses/Unexpected"
+    patch:
+      operationId: patchSignatureRequestStatus
+      summary: Change the status of a signature request
+      parameters:
+        - in: path
+          name: signatureRequestId
+          schema:
+            type: string
+            format: ulid
+          required: true
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/PatchSignatureStatusBody"
+      responses:
+        "204":
+          description: The signature status was changed!
```

Comment:
```suggestion
          description: Signature status successfully changed.
```

Reply:
I'd return the new status

Reply:
ok, maybe it is better to return the entire signature request